app/assets/templates/header_tpl.jst.hbs

```diff
@@ -1,109 +1,121 @@
+              <a href="/notifications" id="view_all_notifications">
+                {{t "header.view_all"}}
+              </a>
+            </div>
+          </ul>
 
-      <div id="notification_dropdown">
-        <div class="header">
-          <div class="pull-right">
-            <a href="#" id="mark_all_read_link" class="{{#unless current_user.notifications_count}}disabled{{/unless}}">
-              {{t "header.mark_all_as_read"}}
-            </a>
-          </div>
+        </li>
 
-          <h4>
-            {{t "header.recent_notifications"}}
-          </h4>
-        </div>
+        <li>
+          <a id="conversations-link" href="/conversations" title="{{t "header.conversations"}}" class="nav-badge">
+            <span class="hidden-xs hidden-sm">
+              <i class="entypo mail"></i>
+            </span>
+            <span class="visible-xs-inline visible-sm-inline">
+              {{t "header.conversations"}}
+            </span>
 
-        <div class="notifications">
-          <div class="ajax_loader">
-            <img class="img-responsive center-block" alt="Ajax-loader" src="{{imageUrl "ajax-loader2.gif"}}">
-          </div>
-        </div>
-        <div class="view_all">
-          <a href="/notifications" id="view_all_notifications">
-            {{t "header.view_all"}}
+            <span class="badge {{#unless current_user.unread_messages_count}} hidden {{/unless}}">
+              {{current_user.unread_messages_count}}
+            </span>
           </a>
-        </div>
-      </div>
-  </div>
+        </li>
+      </ul>
 
-  <ul class="dropdown" id="user_menu">
-    <li class="user-menu-trigger">
-      <div class="user-menu-more-indicator">
-        ▼
-      </div>
-      {{{personImage current_user 'small' 'avatar user-menu-avatar'}}}
-      <a class="user-name" href="#">{{current_user.name}}</a>
-    </li>
-    <li class="user-menu-item"><a href="/people/{{current_user.guid}}">{{t "header.profile"}}</a></li>
-    <li class="user-menu-item"><a href="/contacts">{{t "header.contacts"}}</a></li>
-    <li class="user-menu-item"><a href="/user/edit">{{t "header.settings"}}</a></li>
-    <li class="user-menu-item"><a href="/help">{{t "header.help"}}</a></li>
-    {{#if current_user.admin}}
-      <li class="user-menu-item"><a href="/admins/user_search">{{t "header.admin"}}</a></li>
-    {{/if}}
-    <li class="user-menu-item"><a href="/users/sign_out" data-method="delete">{{t "header.log_out"}}</a></li>
-  </ul>
+      <ul class="nav navbar-nav navbar-right">
+        <li class="dropdown" id="user_menu">
+          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
+            <span class="user-avatar pull-left">
+              {{{personImage current_user "small"}}}
+            </span>
+            <span class="user-name">{{current_user.name}}</span>
+            <span class="caret"></span>
+          </a>
 
+          <ul class="dropdown-menu" role="menu">
```

Comment:
I think you should add `width: 100%;` to that element. It feel weird that the user menu is not the same width than the button.